packages/dart/npt_flutter/lib/features/profile/widgets/profile_popup_menu_button.dart

```diff
@@ -41,10 +43,28 @@ class ProfilePopupMenuButton extends StatelessWidget {
                 }
 
                 if (context.mounted) {
-                  Navigator.of(context).pushNamed(Routes.profileForm, arguments: state.profile.uuid);
+                  Navigator.of(context)
+                      .pushNamed(Routes.profileForm, arguments: ProfileFormPageArguments(state.profile.uuid));
                 }
               },
             ),
+            PopupMenuItem(
```

Comment:
Menu item to duplicate a profile. This operation doesn't need to be supported by multi select, the UX for that would be way too complicated.

Reply:
This is the icon chosen for this item:

![CleanShot 2024-11-19 at 09 02 34@2x](https://github.com/user-attachments/assets/511a6eb6-77bb-4359-a023-d1fb65a8ce70)